src/cf-expandables/usage.md

```diff
@@ -187,15 +187,15 @@ The following combination is our recommended go-to expandable pattern.
     <button class="o-expandable_header o-expandable_target"
             title="Expand content">
         <h3 class="h4 o-expandable_header-left o-expandable_label">
-            Expandable Header
+            Expandable Header that is really really really really really long
```

Comment:
NOTE: I'll revert this after this PR is approved, it's just easier than asking everyone to change the text when testing.